BeeInterns1/index.html

```diff
@@ -0,0 +1,131 @@
+    <link
+      href="https://fonts.googleapis.com/css?family=Abel&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+    <title>BeeInterns</title>
+  </head>
+  <body class="page">
+    <header class="page_header header">
+      <div class="header_logo logo">&nbsp;</div>
+      <nav class="header_nav nav">
+        <div href="#" class="nav_item">&nbsp;</div>
+        <div href="#" class="nav_item">&nbsp;</div>
+        <div href="#" class="nav_item">&nbsp;</div>
+        <div href="#" class="nav_item">&nbsp;</div>
+      </nav>
+    </header>
+    <section class="page_banner banner">
+      <h3 class="banner_title">Banner Header</h3>
+      <h5 class="banner_subtitle">Lorem ipsum dolor sit amet</h5>
+      <div class="banner_btns btns">
+        <button class="btn btn-white">white button</button>
+        <button class="btn btn-black">black button</button>
+      </div>
+    </section>
+
+    <section class="page_articles-row articles">
+      <div class="articles_article article">
+        <div class="article_img img"></div>
+        <div class="article_content">
+          <h3 class="article_header">Article Header</h3>
+          <p class="article_text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation
+          </p>
+          <p class="article_text-footer">Read More</p>
+        </div>
+      </div>
+      <div class="articles_article article">
+        <div class="article_img  img"></div>
+        <div class="article_content">
+          <h3 class="article_header">Article Header</h3>
+          <p class="article_text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation
+          </p>
+          <p class="article_text-footer">Read More</p>
+        </div>
+      </div>
+      <div class="articles_article article">
+        <div class="article_img  img"></div>
+        <div class="article_content">
+          <h3 class="article_header">Article Header</h3>
+          <p class="article_text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation
+          </p>
+          <p class="article_text-footer">Read More</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="page_form form">
+      <h3 class="form_title">Sign Up for updates</h3>
+      <form action="" class="form_inputs">
+        <input type="text" class="form_input" placeholder="Enter you email" />
+        <button class=" form_btn btn btn-black">fill button</button>
+      </form>
```

Comment:
я бы немного поменяла порядок

```suggestion
      <form action="" class="form_inputs">
      <h3 class="form_title">Sign Up for updates</h3>
        <input type="text" class="form_input" placeholder="Enter you email" />
        <button class=" form_btn btn btn-black">fill button</button>
      </form>
```